### ISSUE TITLE
Biz Admin:  Status message after successfully posting offer is red and should be green. 

### ISSUE THREAD

**Issue by MaxTattenbach** [opencurrents/opencurrents#943]
![image](https://user-images.githubusercontent.com/26234440/34440228-613830be-ec79-11e7-979a-2e3032a37838.png)

**Comment by dannypernik** [comment]
@nickolashe I'm interested in knowing how I can fix status message colors. Kostya had defined "msg_type" which is alert if red and success if green. The message displayed in this screenshot comes from OfferCreateView:

```
return redirect(
            'openCurrents:biz-admin',
            'Your offer for %s is now live!' % offer_item.name
        )
```

and in BizAdminView this is the only mention of status_msg:

```
# workaround with status message for anything but TemplateView
        if 'status_msg' in self.kwargs and not context['form'].errors:
            context['status_msg'] = self.kwargs.get('status_msg', '')

        return context
```

**Comment by nickolashe** [comment]
@dannypernik what happens with TemplateView?

**Comment by nickolashe** [comment]
@tevtonez FYI

**Comment by dannypernik** [comment]
@nickolashe "Renders a given template, with the context containing parameters captured in the URL." (https://docs.djangoproject.com/en/2.0/ref/class-based-views/base/#templateview and here)

See question in comment on #919

**Comment by tevtonez** [comment]
this should be fixed in PR #964

**Comment by MaxTattenbach** [comment]
@tevtonez  can't test because of #968.  Seems that the new default on status message could be causing the error.
